modelservice/docker/Dockerfile

```diff
@@ -13,9 +13,9 @@
 # limitations under the License.
 
 FROM openjdk:8-jre
-MAINTAINER Haystack <haystack@expedia.com>
+MAINTAINER Adaptive Alerting <adaptive-alerting@expedia.com>
```

Comment:
just checking this is a live email :)

Reply:
It will be shortly--just requested the DL. Will ensure public as well.